docs/getting-started/installation.md

```diff
@@ -231,33 +231,51 @@ sudo apt update
 
 ### Installing via .rpm Package
 
-**Note:** RHEL, Fedora, and related distributions package QEMU, which Rancher Desktop
-uses on Linux, differently than other distributions. To use Rancher Desktop on these
-distributions, please use the AppImage.
-
+#### openSUSE
```

Comment:
For consistency with the rest of the docs there should always be an empty line after a heading line.